tex/gregoriotex-main.tex

```diff
@@ -238,7 +238,7 @@
 }%
 
 \def\GreFinalNewLine{%
```

Comment:
Was this previously a no-op?  Perhaps we should mark this macro for removal altogether.

Reply:
Yes, it was previously a no-op, generated only when the final break follows a bar. I think it might make sense to keep it or to remove it, but in any case I think that final breaks should be treated the same whether or not they follow a bar.